cmd/skopeo/inspect.go

```diff
@@ -56,9 +57,13 @@ var inspectCmd = cli.Command{
 		if err != nil {
 			return err
 		}
+		var tag string
+		if tagged, ok := img.Reference().DockerReference().(reference.NamedTagged); ok {
+			tag = tagged.Tag()
+		}
 		outputData := inspectOutput{
 			Name: "", // Possibly overridden for a docker.Image.
-			Tag:  imgInspect.Tag,
+			Tag:  tag,
```

Comment:
If it's tagged then tag cannot not be empty, if it's a digest then this code isn't hit and you get an empty tag below in the response which is wanted because an image can't be both tagged and with a digest